WordPressAuthenticator.podspec

```diff
@@ -1,6 +1,8 @@
+# frozen_string_literal: true
+
 Pod::Spec.new do |s|
   s.name          = 'WordPressAuthenticator'
-  s.version       = '1.41.0'
+  s.version       = '1.42.0'
```

Comment:
The ethernal question about "should a `s.swift_version` bump from `4.2` to `5.0` warrant a major or minor version bump in the library?"… (never had a clear answer about this)

Given this lib is mostly used internally I think it's ok, but 🤷